src/main/java/core/basesyntax/service/FruitService.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.activity.ActivityHandler;
+import core.basesyntax.strategy.ActivityStrategy;
+import java.util.List;
+
+public class FruitService {
+    private ActivityStrategy activityStrategy;
+
+    public FruitService(ActivityStrategy activityStrategy) {
+        this.activityStrategy = activityStrategy;
+    }
+
+    public void makeReport(List<FruitTransaction> fruitTransactionList) {
```

Comment:
FruitService and makeReport it's unmatched for me, better call report service or smth else